### ISSUE TITLE
Arrow in Language/Encoding Selection Dropdown Menus covering text

### ISSUE THREAD

**Issue by Ledmington** [texstudio-org/texstudio#1998]
### Environment
- TeXstudio: 4.1.1
- Qt: 6.2.1
- OS: Windows 10
- TeX distribution: texlive

### Expected behavior
The arrow in each one of the drop-down menus should not be over the text.

### Actual behavior
![texstudio dropdown menus](https://user-images.githubusercontent.com/68538713/145027856-50b1218a-2b81-44de-a1f2-7c304c3b4ce0.png)

### How to reproduce
I don't know if this is a real bug or just a configuration issue.

**Comment by sunderme** [comment]
Orion Dark issue

**Comment by Ledmington** [comment]
> Orion Dark issue

Yes, i didn't try at first but changing the style has "solved" the issue. Every other style has the arrows in the correct position or doesn't have them at all.
However, i tried this on another pc of mine and this issue is not present.
Other pc's info:
TeXstudio: 4.1.1
Qt: 5.15.2
OS: Ubuntu 21.10
TeX distribution: texlive

**Comment by Lleafll** [comment]
Seems to be this Qt6 bug: https://bugreports.qt.io/browse/QTBUG-98286
Looks to me like it is going to be fixed in 6.2.3.

**Comment by paksnuk** [comment]
I still observe this issue on TeXstudio 4.2.3 (Qt 6.2.3) on MacOS regardless of the chosen style.
